estate/models/estate_property_offer.py

```diff
@@ -6,12 +6,13 @@
 
 class EstatePropertyOffer(models.Model):
     _name = 'estate.property.offer'
+    _order = 'price DESC'
 
     property_id = fields.Many2one(comodel_name="estate.property", required=True)
     price = fields.Float()
     status = fields.Selection(selection=[('accepted', "Accepted"), ('refused', "Refused")], copy=False)
     partner_id = fields.Many2one(comodel_name="res.partner", required=True)
-
```

Comment:
Nitpicking: this empty line could be removed in an earlier commit...

but really your PR is super clean, I just say this so that you notice I follow up :wink:
Checked until the `[ADD] estate_account` commit.

Nice job !